backend/src/main/java/dev/k2/validateXml/ValidationResponse.java

```diff
@@ -10,4 +10,12 @@ public class ValidationResponse {
     private Boolean valid;
     private List<String> errorMessages;
 
+    public ValidationResponse(Boolean valid, List<String> errorMessages) {
+        this.valid = valid;
+        this.errorMessages = errorMessages;
+    }
+
+    public Boolean getValid() {
```

Comment:
There's `@Data` annotation on this class - it contains getter generation. See [here](https://projectlombok.org/features/Data).